### PR TITLE
Add "Copyright 2021-present" to all file headers

### PR DESCRIPTION
**Pull request by chaoyli**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 StarRocks
 
-Copyright 2021 StarRocks Limited
+Copyright 2022 StarRocks Limited
```
Comment (imay) [inline NOTICE.txt:3]: Should be 2021 - present other than 2022

Comment (chaoyli) [reply NOTICE.txt]: OK

**Comment by chaoyli** [comment]
run starrocks_be_unittest

**Comment by chaoyli** [comment]
run starrocks_be_unittest

**Review by decster — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Pull request merged**